RELEASE-NOTES.md

```diff
@@ -18,6 +18,8 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
   parameter of blt_add_executable
 - Added variable BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE that will be used to filter
   implicit link directories for all languages
+- Added user option for enforcing specific versions of autoformatters - the new options are
+  ``BLT_REQUIRED_ASTYLE_VER``, ``BLT_REQUIRED_CLANGFORMAT_VER``, and ``BLT_REQUIRED_UNCRUSTIFY_VER``
```

Comment:
I think on a whole (though I'm not saying we are perfect), BLT generally doesn't shorten variables names.  Can you change this to VERSION not VER?